step.yml

```diff
@@ -23,6 +23,13 @@ inputs:
       description: Your access token for the account that has access to the Bitrise app.
       is_required: true
       is_sensitive: true
+  - verbose_log: "no"
+    opts:
+      title: Enable verbose logging?
```

Comment:
title says verbose and summary says debug

Reply:
It means the same, summary is just longer.